gqlgenUtils/writeModelUtils.go

```diff
@@ -20,145 +21,286 @@ func WriteModelFiles(modelName string, fields, fieldTypes, files []string) error
 
-				_, err = openFile.WriteString(fmt.Sprintf(`,
-	%s: { type: %s }`, field, fieldTypes[idx]))
+			source := `import { getNode }  from '@server/gql/node';
+import { GraphQLID, GraphQLNonNull, GraphQLObjectType{{#if graphqlInt}}, {{graphqlInt}}{{/if}}{{#if graphqlString}}, {{graphqlString}}{{/if}} } from 'graphql';
+import { getQueryFields, TYPE_ATTRIBUTES } from '@server/utils/gqlFieldUtils';
 
-				if err != nil {
-					return err
-				}
-			}
-			_, err = openFile.WriteString(fmt.Sprintf(`
+const { nodeInterface } = getNode();
+
+export const {{singularModel}}Fields = {
+	id: { type: new GraphQLNonNull(GraphQLID) }{{fieldsWithType fields fieldTypes}}
 };
 
-export const GraphQL%s = new GraphQLObjectType({
-	name: '%s',
+export const GraphQL{{titleSingularModel}} = new GraphQLObjectType({
+	name: '{{titleSingularModel}}',
 	interfaces: [nodeInterface],
 	fields: () => ({
-		...getQueryFields(%sFields, TYPE_ATTRIBUTES.isNonNull)
+		...getQueryFields({{singularModel}}Fields, TYPE_ATTRIBUTES.isNonNull)
 	})
 });
-`,titleSingularModel, titleSingularModel, pluralModel))
+`
+			tpl, err := raymond.Parse(source)
+			if err != nil {
+				return err
+			}
+			result, err := tpl.Exec(ctx)
+			if err != nil {
+				return err
+			}
+			// Opening file by READ & WRITE permission.
+			openFile, err := os.OpenFile(fmt.Sprintf("%s/%s", path, file), os.O_RDWR, 0644)
+			if err != nil {
+				return err
+			}
+			defer openFile.Close()
 
+			_, err = openFile.WriteString(result)
+			if err != nil {
+				return err
+			}
 			err = openFile.Sync()
 			if err != nil {
 				return err
 			}
 			fmt.Printf("%s file updated successfully. \n", file)
 		}
 
+		// Writing into list.js file.
 		if file == "list.js" {
 
+			source := `import { createConnection } from 'graphql-sequelize';
+import db from '@database/models';
+import { GraphQL{{titleSingularModel}} } from './model';
+import { sequelizedWhere } from '@server/database/dbUtils';
+import { totalConnectionFields } from '@server/utils';
+
+export const {{singularModel}}Connection = createConnection({
+	name: '{{pluralModel}}',
+	target: db.{{pluralModel}},
+	nodeType: GraphQL{{titleSingularModel}},
+	before: (findOptions, args, context) => {
+		findOptions.include = findOptions.include || [];
+		findOptions.where = sequelizedWhere(findOptions.where, args.where);
+		return findOptions;
+	},
+	...totalConnectionFields
+});
+`
+			tpl, err := raymond.Parse(source)
+			if err != nil {
+				return err
+			}
+			result, err := tpl.Exec(ctx)
+			if err != nil {
+				return err
+			}
+			openFile, err := os.OpenFile(fmt.Sprintf("%s/%s", path, file), os.O_RDWR, 0644)
+			if err != nil {
+				return err
+			}
+			defer openFile.Close()
+
+			_, err = openFile.WriteString(result)
+			if err != nil {
+				return err
+			}
+			err = openFile.Sync()
+			if err != nil {
+				return err
+			}
+			fmt.Printf("%s file updated successfully.\n", file)
 		}
+
+		// writing into query.js file.
 		if file == "query.js" {
```

Comment:
lets move this into a handle bar file. so we just read from there.

Reply:
Yes.